includes/ApplHandlingByStr.php

```diff
@@ -82,6 +86,22 @@ function showFullStatus( $stat_str )
 }
 
 
+function findPending( $stat_str )
```

Comment:
__[code_sniffer]__
* Expected 0 spaces between opening bracket and argument "$stat_str"; 1 found
* Expected 0 spaces between argument "$stat_str" and closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->